samples/net/sockets/echo_client/overlay-ot.conf

```diff
@@ -0,0 +1,39 @@
+CONFIG_CPLUSPLUS=y
+
+CONFIG_BT=n
+
+# Disable TCP and IPv4
+CONFIG_NET_TCP=n
```

Comment:
Why disable TCP?

Reply:
TCP in these samples produces quite heavy traffic which results with some packet loss on the UDP side. Therefore I'd rather avoid to use both.